### PR TITLE
Normalize configure script

### PR DESCRIPTION
**Pull request by sl1pkn07**

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -80,21 +81,17 @@ TARGET_OS=""
...
-
 DEBUG=""
 
 LIBNAME=""
 
 CXXFLAGS="-std=c++11 -Wall -I. -I$SRCDIR/src"
 LDFLAGS=""
-SOFLAGS="-shared -Wl,-Bsymbolic"
-DEPLIBS=""
+SOFLAGS="-shared"
```
Comment (sl1pkn07) [inline configure:94]: opts. the original have -Wl,-Bsymbolic, is still needed?

Comment (msg7086) [inline configure:94]: I assume ... not?

Comment (sl1pkn07) [inline configure:94]: then change please. this is a typo by my side when compare with other configure without that flag

note:https://software.intel.com/en-us/articles/performance-tools-for-software-developers-bsymbolic-can-cause-dangerous-side-effects

Comment (msg7086) [inline configure:94]: So this patch is good?

Comment (sl1pkn07) [inline configure:94]: Really, I don't know

**Pull request merged**